docs/configuration.rst

```diff
@@ -8,14 +8,25 @@ Environment variables
 -------------------------
 We use environment variables for:
 
-* Configuration
+* Configuration (mapview)
+* Credentials
 
-  * choosing a mapview backend that delivers the map tiles (square pngs of the map at a certain zoom level) to the user. For more technical information, refer to :ref:`architecture:mapview`.
+This is a complete list of environment variables we use. Their documentation is documented further in their respective topic:
```

Comment:
Is it though? Do we use more?

Reply:
I cannot think of any other variables except the ones used in travis testing logic